README.md

```diff
@@ -33,12 +33,24 @@ Here is an example file...
 
 You can use the **Gitpod** badge or to do it manually...
 
-1. Fork this repo
+1. Fork [**this**](https://github.com/EddieHubCommunity/EventCalendar) repository
```

Comment:
I'm not sure of the purpose of a link here when you are already on this page.

Reply:
Yes that's right, I'll remove it.